ui/app/templates/partials/role-aws/form.hbs

```diff
@@ -3,10 +3,11 @@
     {{message-error model=model}}
     <NamespaceReminder @mode={{mode}} @noun="AWS role" />
     {{#if (gt model.credentialTypes.length 1)}}
-      {{#message-in-page type="warning"}}
-        This role has more than one <code>credential_type</code>, currently: <code>{{join ', ' model.credentialTypes}}</code>.
-        Multiple credential types is deprecated and you must choose one in order to save this role.
-      {{/message-in-page}}
+      <AlertBanner
+        @type="warning"
+        @message="This role has more than one <code>credential_type</code>, currently: <code>{{join ', ' model.credentialTypes}}</code>.
```

Comment:
Same here as above, that code will be in the output - was there a reason this moved to an attribute instead of using the block form?

Reply:
So these <code> bits will have to come out.